scripts/create-cat-api.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+#######################
+# CaT API - TODO: Rely on deployment via travis only / in certain envs?
+#######################
+  # cf push -k $DISK_API -m $MEMORY_API -i $INSTANCES_API $APP_NAME \
+  #   --docker-image $DOCKER_IMAGE_SPREE --docker-username $AWS_ECR_REPO_ACCESS_KEY_ID \
+  #   -c "bundle exec sidekiq" --no-start --no-route -u process
+
+  # Map an internal route to CaT API backend for UI
+  # cf map-route $APP_NAME apps.internal --hostname $APP_NAME
+fi
```

Comment:
Looks like a straggler?

Reply:
Oops - yep 👍 
(Haven't run this one yet!)